src/Microsoft.ML.PipelineInference/Macros/PipelineSweeperMacro.cs

```diff
@@ -36,6 +36,33 @@ public sealed class Arguments
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Output datasets from previous iteration of sweep.", SortOrder = 7, Hide = true)]
             public IDataView[] CandidateOutputs;
+
+            [Argument(ArgumentType.MultipleUnique, HelpText = "Column(s) to use as purpose 'Ignore'", SortOrder = 8, Hide = true)]
```

Comment:
>Hide = true [](start = 117, length = 11)

why you hide them?

Reply:
Any thoughts on what's the implication of the 'Hide' argument for a Macro?  Wasn't clear to me.

---
In reply to: [199589626](https://github.com/dotnet/machinelearning/pull/461#discussion_r199589626) [](ancestors = 199589626)